apps/backoffice-v2/src/common/components/molecules/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,22 @@
+import { DropdownMenu } from '@/common/components/molecules/DropdownMenu/DropdownMenu';
+import { DropdownMenuTrigger } from '@/common/components/molecules/DropdownMenu/DropdownMenu.Trigger';
+import { DropdownMenuContent } from '@/common/components/molecules/DropdownMenu/DropdownMenu.Content';
+import { isNonEmptyArray } from '@ballerine/common';
+import { DropdownMenuItem } from '@/common/components/molecules/DropdownMenu/DropdownMenu.Item';
+import React from 'react';
+import { DropdownProps } from '@/common/components/molecules/Dropdown/interfaces';
+
+export const Dropdown = <TItem,>({ options, trigger, props, children }: DropdownProps<TItem>) => {
```

Comment:
we dont have any existing dropdown components already?

Reply:
It was requested that we'll start creating components that abstract ShadCN's multiple pieces. The dropdown we used until now was `DropdownMenu.Item`, `DropdownMenu.Content` and so on inline.